v63000/u_inref/gtm8394.csh

```diff
@@ -14,6 +17,21 @@
 # GTM-8394 [nars] GTMASSERT in mur_insert_prev.c line 148 using V62000 after GTM-F-MEMORY interrupted rollbacks
 #
 
+# This test runs with limited memory setting and glibc does dlopen() of certain libraries (e.g. libgcc_s.so.1) on the fly
+# (instead of opening them as part of the dlopen of libc.so) and so it is possible if/when those libraries are needed
+# (e.g. pthread_cancel is one place where we have seen it), we might not have enough memory for the dlopen. In that case,
+# the following message is issued by libc.
+#
+#	libgcc_s.so.1 must be installed for pthread_cancel to work
+#
+# This message is issued on the terminal where this test was issued from even if the test stdout/stderr have been
+# redirected to a file. This is because the above message is treated as a fatal libc error and those go through
+# a special function that tries to open the terminal (if available) to log the error. The following env var
+# overrides that logic and continue to log this error in stderr.
+#
+setenv LIBC_FATAL_STDERR_ 1
+set gccabort = "libgcc_s.so.1 must be installed for pthread_cancel to work"
```

Comment:
My concern here is grep-ping for a specific error that includes a specific version of a libgcc library. At any time this library level could be different - and could potentially even be different on different distros. The message could change too but that's less likely than the library version. Perhaps wildcard the version id?